scripts/rl_conclusion_registry.py

```
@@ -0,0 +1,192 @@
+        incoming_owner = incoming.get("ownerCron")
+        if isinstance(incoming_owner, str) and incoming_owner != owner_cron:
+            raise ConclusionRegistryError(
+                f"conclusion {conclusion_id} is owned by {incoming_owner}, not producer {owner_cron}"
+            )
+
+        previous = existing_records.get(conclusion_id)
+        previous_owner = previous.get("ownerCron") if isinstance(previous, dict) else None
+        if previous is not None and previous_owner != owner_cron:
+            raise ConclusionRegistryError(
+                f"refusing to overwrite conclusion {conclusion_id} owned by {previous_owner!r}"
+            )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)